src/cards/ha-badges-card.html

```diff
@@ -4,6 +4,12 @@
 
 <dom-module id='ha-badges-card'>
   <template>
+    <style>
+      ha-state-label-badge {
+        display: inline-block;
+        margin-bottom: var(--ha-state-label-badge-margin-bottom, 16px);
```

Comment:
Same idea, does this card need to define the margin? 😛  feels like something that should be done by the component that actually cares about this margin.